packages/rum-core/src/domain/resource/resourceCollection.spec.ts

```diff
@@ -42,14 +43,12 @@ describe('resourceCollection', () => {
 
   it('should create resource from performance entry', () => {
     const { lifeCycle, rawRumEvents } = setupBuilder.build()
-    const performanceEntry = createResourceEntry({
-      duration: 100 as Duration,
-      name: 'https://resource.com/valid',
-      startTime: 1234 as RelativeTime,
-    })
+
+    const performanceEntry = createPerformanceEntry(RumPerformanceEntryType.RESOURCE)
+
     lifeCycle.notify(LifeCycleEventType.PERFORMANCE_ENTRIES_COLLECTED, [performanceEntry])
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏spaces needed?
```suggestion
    const performanceEntry = createPerformanceEntry(RumPerformanceEntryType.RESOURCE)
    lifeCycle.notify(LifeCycleEventType.PERFORMANCE_ENTRIES_COLLECTED, [performanceEntry])
```

Reply:
Sure